datahub-web-react/src/app/entity/user/UserEditProfileModal.tsx

```diff
@@ -0,0 +1,221 @@
+import React, { useState } from 'react';
+import { message, Button, Input, Modal, Typography, Form } from 'antd';
+import { useUpdateCorpUserPropertiesMutation } from '../../../graphql/user.generated';
+
+type Props = {
+    visible: boolean;
+    onClose: () => void;
+    onSave: () => void;
+    data: {
+        name: string | undefined;
+        title: string | undefined;
+        image: string | undefined;
+        team: string | undefined;
+        email: string | undefined;
+        slack: string | undefined;
+        phone: string | undefined;
+        urn: string | undefined;
+    };
+};
+/** Regex Validations */
+export const validUserName = new RegExp('^[a-zA-Z ]*$');
+
+export default function UserEditProfileModal({ visible, onClose, onSave, data }: Props) {
+    const [updateCorpUserPropertiesMutation] = useUpdateCorpUserPropertiesMutation();
+    const [form] = Form.useForm();
+
+    const [userName, setUserName] = useState<string | undefined>(data.name);
+    const [userTitle, setUserTitle] = useState<string | undefined>(data.title);
+    const [userImageURL, setImageURL] = useState<string | undefined>(data.image);
+    const [userTeam, setUserTeam] = useState<string | undefined>(data.team);
+    const [userEmail, setUserEmail] = useState<string | undefined>(data.email);
+    const [userSlack, setUserSlack] = useState<string | undefined>(data.slack);
+    const [userPhoneNumber, setUserPhoneNumber] = useState<string | undefined>(data.phone);
+    const [buttonDisabled, setButtonDisabled] = useState(true);
+
+    // save changes function
+    const onCreateGroup = () => {
```

Comment:
onSaveChanges?

Reply:
(Rename this function)

Reply:
The function name change  from `onCreateGroup`  to `onSaveChanges`.